docs/development/design/new-sphinx-guides.rst

```diff
@@ -73,23 +73,24 @@ highly recommended.
       -  Open http://localhost:8000 in a web browser, and see the HTML
          documentation (yay!)
 
-   2. Simplify documentation building by using Make [3]_
+   2. Converting our documentation to other formats
+
+      -  Install TeX (MiKTeX looks very straightforward on Windows, TeX Live
+         is an alternative)
+      -  Build PseudoXML using ``make pseudoxml``
```

Comment:
This is a particularly neat one, because it shows the docutils AST. Could be worth diving into in some more depth in an appendix.

Reply:
Cool! Added an appendix about it 👍🏽